.teamcity/_self/projects/WebApp.kt

```diff
@@ -214,6 +214,19 @@ object RunCalypsoE2eMobileTests : BuildType({
 			"""
 			dockerImage = "%docker_image_e2e%"
 		}
+		bashNodeScript {
+			name = "Update Chrome"
+			scriptContent = """
+				export CHROME_VERSION="90.0.4430.93-1"
+				wget --no-verbose https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}_amd64.deb
+				apt-get install -y ./google-chrome-stable_${CHROME_VERSION}_amd64.deb
```

Comment:
TeamCity needs to escape `$`, the correct syntax is `${'$'}CHROME_VERSION`

Reply:
Hah yea, first time with Kotlin, inlined the version, we can move it out for the docker file change.